frontend/src/metabase/admin/permissions/components/PermissionsSidebar/PermissionsSidebar.tsx

```diff
@@ -16,7 +16,7 @@ export const PermissionsSidebar = ({
   isLoading,
   error,
   ...contentProps
-}: PermissionsSidebar) => {
+}: PermissionsSidebarProps) => {
```

Comment:
ESLint seems to have trouble when both types and variables are the same names. However, it's a good practice to differentiate between type and component anyway.